src/data/polynomial/laurent.lean

```diff
@@ -356,6 +360,111 @@ begin
     simpa using hn }
 end
 
+section support
+
+lemma support_C_mul_T (a : R) (n : ℤ) : (C a * T n).support ⊆ {n} :=
+by simpa only [← single_eq_C_mul_T] using support_single_subset
+
+lemma support_C_mul_T_ne_zero {a : R} (a0 : a ≠ 0) (n : ℤ) : (C a * T n).support = {n} :=
+begin
+  rw ← single_eq_C_mul_T,
+  exact support_single_ne_zero _ a0,
+end
+
+@[simp] lemma to_laurent_support (f : R[X]) :
```

Comment:
Is the RHS really simpler than the LHS?

Reply:
My reasoning is that, even though the RHS has more characters, it is simply taking the image of a finset under `nat_cast` into `with_bot int`.  The LHS involves knowing what `to_laurent` is and, in particular, it involves Laurent polynomials.  In the RHS, Laurent polynomials have disappeared entirely.

Does this seem like a good enough reason/justification?

Reply:
`simp` tag removed and doc-string added.  Let me know if you have further comments/changes!

And thank you very much for the review